packages/launchpad/src/components/code/FileRow.vue

```diff
@@ -54,7 +54,7 @@
       :code="content"
       line-numbers
       copy-on-click
-      copy-button
+      :copy-button="copyButton"
```

Comment:
How about we don't set this at all? There's no instance of FileRow that's receiving the prop, is it something FileRow needs to know how to handle? I might just not be seeing it.